---
description: '詳細情報: System.IdentityModel.IdentityModelAsyncCallbackThrewException'
title: System.IdentityModel.IdentityModelAsyncCallbackThrewException
ms.date: 03/30/2017
ms.assetid: e4fc924f-3b9e-44f5-97e4-b3a4bb855d89
ms.openlocfilehash: 304cf448b71c6a0aa901a09273b81b686020d0f2
ms.sourcegitcommit: ddf7edb67715a5b9a45e3dd44536dabc153c1de0
ms.translationtype: HT
ms.contentlocale: ja-JP
ms.lasthandoff: 02/06/2021
ms.locfileid: "99635287"
---
# <a name="systemidentitymodelidentitymodelasynccallbackthrewexception"></a><span data-ttu-id="3d7b1-103">System.IdentityModel.IdentityModelAsyncCallbackThrewException</span><span class="sxs-lookup"><span data-stu-id="3d7b1-103">System.IdentityModel.IdentityModelAsyncCallbackThrewException</span></span>

<span data-ttu-id="3d7b1-104">System.IdentityModel.IdentityModelAsyncCallbackThrewException</span><span class="sxs-lookup"><span data-stu-id="3d7b1-104">System.IdentityModel.IdentityModelAsyncCallbackThrewException</span></span>  
  
## <a name="description"></a><span data-ttu-id="3d7b1-105">説明</span><span class="sxs-lookup"><span data-stu-id="3d7b1-105">Description</span></span>  

 <span data-ttu-id="3d7b1-106">非同期コールバックで例外がスローされました。</span><span class="sxs-lookup"><span data-stu-id="3d7b1-106">An asynchronous callback threw an exception.</span></span>  
  
## <a name="see-also"></a><span data-ttu-id="3d7b1-107">関連項目</span><span class="sxs-lookup"><span data-stu-id="3d7b1-107">See also</span></span>

- [<span data-ttu-id="3d7b1-108">トレース</span><span class="sxs-lookup"><span data-stu-id="3d7b1-108">Tracing</span></span>](index.md)
- [<span data-ttu-id="3d7b1-109">トレースを使用したアプリケーションのトラブルシューティング</span><span class="sxs-lookup"><span data-stu-id="3d7b1-109">Using Tracing to Troubleshoot Your Application</span></span>](using-tracing-to-troubleshoot-your-application.md)
- [<span data-ttu-id="3d7b1-110">管理と診断</span><span class="sxs-lookup"><span data-stu-id="3d7b1-110">Administration and Diagnostics</span></span>](../index.md)
